lib/react_client.dart

```diff
@@ -476,72 +476,24 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
       handleRender: allowInterop(handleRender));
 })();
 
-// TODO custom adapter for over_react to avoid typedPropsFactory usages?
-class JsComponent2Adapter extends Component2Adapter {
-  // TODO find a way to inject this better
-  final ReactComponent jsThis;
+abstract class _ReactDartInteropStatics2 {
+  // TODO 3.1.0-wip expose for testing?
+  static final zone = Zone.root;
 
-  JsComponent2Adapter({
-    this.jsThis,
-  });
-
-  @override
-  void forceUpdate(SetStateCallback callback) {
```

Comment:
These adapter method implementations are pretty much unchanged, and just moved to bridge.dart